airflow/cli/commands/remote_commands/task_command.py

```diff
@@ -206,6 +206,11 @@ def _get_ti(
     dag = task.dag
     if dag is None:
         raise ValueError("Cannot get task instance for a task not assigned to a DAG")
+    if not isinstance(dag, DAG):
+        # SHouldn't really happen, and this command will go away before 3.0
```

Comment:
(1) typo... and (2) should it have a TODO to be remonded to clean-up before 3.0? 
```suggestion
        # Shouldn't really happen, and TODO(ashb) this command will go away before 3.0
```